app/adapters/osf-adapter.ts

```diff
@@ -24,13 +19,14 @@ const { JSONAPIAdapter } = DS;
  * @extends DS.JSONAPIAdapter
  * @uses GenericDataAdapterMixin
  */
-export default class OsfAdapter extends JSONAPIAdapter.extend(GenericDataAdapterMixin, {
+export default class OsfAdapter extends JSONAPIAdapter.extend(GenericDataAdapterMixin).extend({
```

Comment:
Double extend? I haven't seen it before. Maybe add a comment for why?

Reply:
Hm, for some reason I thought that was required for typescript to get the types from the mixin properly, but that's not true. Reverting...